scripts/changelog-links.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash
```

Comment:
so this is an interesting one..this script is usually run by the TeamCity configuration? perhaps to re-use this behavior, an additional job in the `release.yml` can run this  script and commit the updated changelog file🤔 in the `awscc` provider i guess we haven't run into many situations where the changelog entries are referencing specific PRs so that part has been pretty manual

Reply:
ah! gotcha, i just read in the RFC that we could remove this. for tpg/tpgb we generate our changelog entries anyway. i'll have to look at that changelog generation to see if our changelog generates to the `[GH-nnnn]` format or just includes the links? I'd assume the latter, but will double-check! Thanks!